### PR TITLE
[zh] sync changes in docs/reference/setup-tools/kubeadm/generated/ directory

### PR DESCRIPTION
**Pull request by howieyuen**
ref: https://github.com/kubernetes/website/issues/25247
sync files below, 22 in total
```
1	1	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_token_list.md
2	2	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-join_etcd.md
2	2	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-prepare_all.md
1	1	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_list.md
10	2	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node.md
3	3	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md
0	14	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_etcd-healthcheck-client.md
0	7	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane_scheduler.md
0	7	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane_all.md
0	14	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver-kubelet-client.md
2	2	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_control-plane.md
0	14	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_etcd-peer.md
0	7	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane_apiserver.md
2	2	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-prepare_control-plane.md
0	14	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver.md
0	7	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane_controller-manager.md
2	9	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md
2	2	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_etcd_local.md
0	14	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver-etcd-client.md
0	14	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_front-proxy-client.md
2	2	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md
0	14	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_etcd-server.md
```

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md [hunk 1]
```diff
@@ -241,14 +243,18 @@ For token-based discovery, allow joining without --discovery-token-ca-cert-hash
...
-The path where kustomize patches for static pod manifests are stored.
+<!--  
+Path to a directory that contains files named "target[suffix][+patchtype].extension". For example, "kube-apiserver0+merge.yaml" or just "etcd.json". "patchtype" can be one of "strategic", "merge" or "json" and they match the patch formats supported by kubectl. The default "patchtype" is "strategic". "extension" must be either "json" or "yaml". "suffix" is an optional string that can be used to determine which patches are applied first alpha-numerically.
 -->
-用于存储 kustomize 为静态 pod 清单所提供的补丁的路径。
+包含名为 "target[suffix][+patchtype].extension" 的文件的目录的路径。
+例如，"kube-apiserver0+merge.yaml" 或仅仅是 "etcd.json"。
+"patchtype" 可以是 "strategic"、"merge" 或 "json" 之一，并且它们与 kubectl 支持的补丁格式匹配。
+默认的 "patchtype" 为 "strategic"。 "extension" 必须为 "json" 或 "yaml"。 
+"suffix" 是一个可选字符串，可用于确定首先按字母数字方式应用哪些补丁。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md:257]:
```suggestion
"suffix" 是一个可选字符串，可用于确定首先按字母顺序应用哪些补丁。
```

**Review by tengqm — COMMENTED** [review]
LGTM
just one nit

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md (hunk 1 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md]: 已修改

**Comment by tengqm** [comment]
THANKS
/lgtm
/approve

**Pull request merged**